manifests/keyboard/capslock_to_no_action.pp

```diff
@@ -0,0 +1,16 @@
+# Public: Remaps capslock to no action on attached keyboards.
+#
+# Example
+#
+#   include osx::keyboard::capslock_to_no_action
+#
+class osx::keyboard::capslock_to_no_action {
+  # Remap capslock to no action on all attached keyboards
+  $keyboard_ids = 'ioreg -n IOHIDKeyboard -r | grep -E \'VendorID"|ProductID\' | awk \'{ print $4 }\' | paste -s -d\'-\n\' -'
+  $check = 'xargs -I{} sh -c \'defaults -currentHost read -g "com.apple.keyboard.modifiermapping.{}-0" | grep "Dst = 2" > /dev/null\''
```

Comment:
This check is not quite correct. You want to check if the value is already set to `"-1"` (no action) and not 2 (control).